pierone/cli.py

```diff
@@ -93,6 +105,7 @@ def set_pierone_url(config: dict, url: str) -> None:
         # issue 63: gracefully handle URLs without scheme
         url = 'https://{}'.format(url)
```

Comment:
It can start with `http://` as well. It could be `if not re.match('^https?://.*', url)`, but the current form is simpler and do the work.